api/swagger-spec/storage.k8s.io_v1beta1.json

```diff
@@ -675,7 +675,7 @@
      },
      "metadata": {
       "$ref": "v1.ObjectMeta",
-      "description": "Standard object's metadata. More info: http://releases.k8s.io/HEAD/docs/devel/api-conventions.md#metadata"
```

Comment:
We used to have a script which would replace HEAD in `http://releases.k8s.io/HEAD/docs` by the release version in our generated docs on k8s.io. For ex: replace http://releases.k8s.io/HEAD/docs by http://releases.k8s.io/v1.6/docs.
We will need to update that script as well?